frontend/src/components/particles/CredentialHeader.vue

```diff
@@ -59,7 +50,7 @@
             />
           </ValidationProvider>
         </template>
-        <v-date-picker v-model="main.dob" scrollable>
+        <v-date-picker v-model="main.dob" scrollable max="2021-01-20">
```

Comment:
Probably cleaner to set it to today's date rather than hardcoded to a certain date